packages/pigeon/lib/swift_generator.dart

```diff
@@ -338,7 +338,7 @@ void _writeFlutterApi(Indent indent, Api api, Root root) {
             indexMap(func.arguments, _getArgumentName);
         final Iterable<String> argNames =
             indexMap(func.arguments, _getSafeArgumentName);
-        sendArgument = '[${argNames.join(', ')}]';
+        sendArgument = '[${argNames.join(', ')}] as [Any?]';
```

Comment:
This fixes the warning exposed by the multiple-argument Flutter API method I added to core_tests.dart. The problem is that `[somethingOfTypeA, somethingOfTypeB]` implicitly discards type information about the elements because it becomes an `[Any]` (or `[Any?]`). You have to make it explicit that you know you are doing that to silence the warning.